backend/src/main.py

```diff
@@ -318,6 +334,49 @@ def extract_graph_from_file(uri, userName, password, model, file=None,s3_url=Non
       logging.exception(f'Exception Stack trace:')
       return create_api_response(job_status,error=error_message)
 
+def get_documents_from_file(file):
+    print("get_documents_from_file called")
+    file_name = file.filename
+    print("file_name = ",file_name)
+    file_key=file_name
+    print("file_key = ",file_key)
+        
+    with open('temp.pdf','wb') as f:
```

Comment:
how does this work with multiple users importing data at the same time,
you should use temporary file names that are cleaned up automatically or after the loader is done.